targets/xiwi

```diff
@@ -28,8 +28,27 @@ if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
     apt-mark unhold xserver-xorg-video-dummy$ltspackages || true 2>/dev/null
```

Comment:
@drinkcat to add flavor

> Is there a reason why the driver has to overwrite the dummy driver? Why not install it as a different driver (crouton_dummy_drv.so perhaps?) and then change the X configuration to load that driver instead?

Probably for a couple reasons

1. It keeps the patch smaller
2. It's a strict superset of dummy, so it's compatible with anything already trying to use dummy
3. The intent is that the changes will be upstreamed, so we don't want a separate crouton dummy driver hanging around when the patch is no longer necessary

> Is there a reason why the patch is inlined in the shell script rather than sitting in a separate file? I was thinking of applying it as an overlay patch to the Gentoo package, which would also resolve the issue of the driver being overwritten.

Probably because there's no clean way to do that in crouton at the moment, and we didn't want to leave files lying around.  An overlay patch in Gentoo would make a lot of sense, so maybe in this case, add the patch to chrootetc?  Then the prepare script can run patch on that file, and you can symlink it into the local Gentoo overlay.